.github/workflows/ci-tests.yml

```diff
@@ -203,7 +203,7 @@ jobs:
       # Do not set explicitly temp dir for dbt as causes problems
       # None of these test need temp dir set
       run: |
-        python -m tox -e dbt018-winpy -- plugins/sqlfluff-templater-dbt -m "not dbt_connection_failure and not needs_dbt_connection"
+        python -m tox -e dbt018-winpy -- plugins/sqlfluff-templater-dbt -m "not needs_dbt_connection"
```

Comment:
dbt_connection_failure is mocked so never actually connects to db, let's try not excluding

Reply:
okay CI shows that this test still connects to the db, we just mock the return value to pretend it fails. Initial proposal is right. Removed.

Reply:
Good shout.

I also think I also think I added one too many:

![image](https://user-images.githubusercontent.com/10931297/148612547-8000181e-0e17-4c2d-9960-70168af54edf.png)

Would you mind reverting that as part of this PR? Think I read it as "DB Error" instead of "DBT Error" and thought it was needed here but don't think it is.

Reply:
no worries will do 😄